paddle/framework/backward.cc

```diff
@@ -91,7 +89,7 @@ std::shared_ptr<OperatorBase> BackwardRecursive(
   }
 
   // Returned gradient network
-  auto net = std::make_shared<operators::NetOp>();
+  auto net = std::unique_ptr<operators::NetOp>(new operators::NetOp());
```

Comment:
Well, it could be in another PR. `make_unique` is not in `C++ 11` but in `C++ 14`. We can implement it in `C++ 11`.